api/v1/aerospikecluster_validating_webhook.go

```diff
@@ -2282,3 +2306,61 @@ func (c *AerospikeCluster) validateMaxUnavailable() error {
 
 	return nil
 }
+
+func (c *AerospikeCluster) validateEnableDynamicConfigUpdate() error {
+	if !GetBool(c.Spec.EnableDynamicConfigUpdate) {
+		return nil
+	}
+
+	if len(c.Status.Pods) == 0 {
+		return nil
+	}
+
+	minInitVersion, err := getMinRunningInitVersion(c.Status.Pods)
+	if err != nil {
+		return err
+	}
+
+	val, err := lib.CompareVersions(minInitVersion, minInitVersion4DynamicConf)
+	if err != nil {
+		return fmt.Errorf("failed to check image version: %v", err)
+	}
+
+	if val < 0 {
+		return fmt.Errorf("cannot enable enableDynamicConfigUpdate flag, init container are running version less than %s",
```

Comment:
```suggestion
		return fmt.Errorf("cannot enable enableDynamicConfigUpdate flag, init containers are running version less than %s",
```
```suggestion
		return fmt.Errorf("cannot enable enableDynamicConfigUpdate flag, init container is running version less than %s",
```

Reply:
Should we also say to do a cold restart to upgrade init version or point to aerospike doc ?